debian/rules

```diff
@@ -59,6 +59,7 @@ override_dh_installinit:
 	install -d -m0755 debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-osd*.conf debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-disk.conf debian/ceph-osd/etc/init
+	install -m0644 build/src/upstart/ceph-osd.conf debian/ceph-osd/etc/init
```

Comment:
It would be nice if we could drop upstart support. Currently we are still building for Trusty, though. [The last time Sage brought it up on the mailing lists](http://lists.ceph.com/pipermail/ceph-users-ceph.com/2016-November/014309.html) I got the sense that Luminous would still be supported there?